nashville-software-school/Class-Website
# Ps photos1

## Description of Proposed Changes

- all photos added

## Steps to Test

make sure all photos and baby photos load.

```sh
git fetch --all
git checkout <ps-photos1>

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* json file for users
* baby photo files
